src/main/grammars/ElmParser.bnf

```diff
@@ -392,7 +392,7 @@ Field ::=
 
 
 IfElse ::=
-    IF Expression THEN Expression ELSE Expression
+    IF Expression THEN Expression (ELSE IF Expression THEN Expression)* ELSE Expression
```

Comment:
This change matches how the Elm compiler parses these expressions, and makes them easier to work with.